pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -284,7 +311,7 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
             />
           ),
 
-          enableNext: !isMutating,
+          enableNext: !isMutating, //&& candidatesApplications.length > 0,
```

Comment:
Is this intended?

Reply:
Yes, I need to add a condition for next. Thanks !